legend-engine-xt-analytics-search-pure/src/main/resources/core_analytics_search/metamodel.pure

```diff
@@ -0,0 +1,104 @@
+Class meta::analytics::search::metamodel::BaseRootDocument
+[
+  idNotBlank: $this.id->length() > 0,
+  typeNotBlank: $this.type->length() > 0,
+  logicalTypeNotBlank: $this.logicalType->length() > 0
+]
+{
+  projectCoordinates : meta::analytics::search::metamodel::ProjectCoordinates[0..1];
+  id: String[1];
+  type: String[1];
+  logicalType: String[1];
```

Comment:
Does this belong in this model, or is it more of a UI concern?

Reply:
Removed logicalType